components/panel-body/style.scss

```diff
@@ -0,0 +1,4 @@
+.components-panel-body {
+	padding: 15px;
+	box-shadow: 0 0 0 1px $light-gray-500;
```

Comment:
At a glance, could we use border here instead? I'm sensitive to become overly reliant on box-shadow as it can have miserable performance.

http://tobiasahlin.com/blog/how-to-animate-box-shadow/
https://medium.com/airbnb-engineering/css-box-shadow-can-slow-down-scrolling-d8ea47ec6867